### PR TITLE
make scheme of live/readiness probes configurable

### PR DESCRIPTION
**Pull request by erohmensing**
Starting pods on https will make them fail their liveness probes; make this configurable via values.

### CODE REVIEW

File: charts/rasa-x/templates/app-deployment.yaml [hunk 1]
```diff
@@ -50,10 +50,12 @@ spec:
           httpGet:
```
Comment (tczekajlo) [inline charts/rasa-x/templates/app-deployment.yaml:50]: Hmm, we can go further and pass the whole spec for liveness probe and readiness probe, e.g.:
```
readinessProbe: 
{{ toYaml. Values.app.readinessProbe }}
```

File: charts/rasa-x/templates/event-service-deployment.yaml [hunk 2]
```diff
@@ -57,10 +57,12 @@ spec:
           httpGet:
             path: "/health"
             port: "http"
+            scheme: {{ default "HTTP" .Values.eventService.probeScheme }}
         readinessProbe:
           httpGet:
             path: "/health"
             port: "http"
+            scheme: {{ default "HTTP" .Values.eventService.probeScheme }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/event-service-deployment.yaml:65]: I think I could make a distinction between probes, e.g.

```
readinessProbe:
  scheme: {{ default "HTTP" .Values.eventService.readinessProbe.scheme }}
livenessProbe:
  scheme: {{ default "HTTP" .Values.eventService.livenessProbe.scheme }}
```

And the same for the rest of the deployments.

**Review by tczekajlo — COMMENTED** [review]
one comment 😄

File: charts/rasa-x/templates/event-service-deployment.yaml (hunk 2 above)
Comment (erohmensing) [reply charts/rasa-x/templates/event-service-deployment.yaml:65]: Sure thing, makes sense!

**Review by tczekajlo — APPROVED** [review]
🚀

**Pull request merged**